incubator/elasticsearch-exporter/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```

Comment:
Use namespaced templates. Creating a chart with `helm create` would give you a good basis with current best practices applied.